discojs/secAggREADME.md

```diff
@@ -0,0 +1,20 @@
+# Secure Multiparty Aggregation for Decentralized Training
+
+### Algorithm description
+Additive sharing (you can read more about this algorithm [HERE](https://www.geeksforgeeks.org/additive-secret-sharing-and-share-proactivization-using-python/?ref=rp)
+) is implemented with each client's gradients when the gradient is sent to all other clients connected in this round of multiparty training. The algorithm has four steps:
+1. Each client will send a "ready message" to the signaling server, indicating that they are done with training and ready to be connected to other clients in order to aggregate weights. Each client pauses aggregation until they have received a list of peers to connect to.
+2. Once the client has a list of peers, they will generate *n* shares, where _n_ is the number of clients (including themself) who will receive a share in order to be a part of this round of secure aggregation. Each client will receive 1 share from each other client. Each client will then have _n_ shares.
```

Comment:
secret share, generated out of the secret sth value (model or model update)

Reply:
say intuitively what this is. the true value additively masked with some secret noise